Documentation/install-rkt-in-coreos.md

```diff
@@ -0,0 +1,31 @@
+# Installing a different version of rkt in CoreOS
+
+If a different version of rkt is required than what ships with CoreOS, a
+oneshot systemd unit can be used to download and install an alternate version
+on boot.
+
+The following unit will use curl to download rkt, a stage1, and a signature for
+the stage1. rkt is then extracted to /opt with the tar command.
+
+```
+[Unit]
+Description=rkt installer
+Requires=network.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/bin/curl -L -o /opt/rkt.tar.gz <rkt-url>
+ExecStart=/usr/bin/curl -L -o /opt/stage1-coreos-linux-amd64.aci <rkt-stage1-url>
+ExecStart=/usr/bin/curl -L -o /opt/stage1-coreos-linux-amd64.aci.asc <rkt-stage1-asc-url>
+ExecStart=/usr/bin/tar xf /opt/rkt.tar.gz -C /opt
```

Comment:
> and the tarball already included the stage1 in any case

Yep, and I believe that the stage1 included in the rkt tar file is the one that will be used. Note that we're not checking the signatures of the rkt tar... 

> A possible solution to this is tar --strip-components=1 -xf rkt.tar.gz -C /opt/rkt/ which leaves you with a consistent /opt/rkt/rkt.

+1